.travis.yml

```diff
@@ -11,7 +11,9 @@ env:
   - RUBY_GC_HEAP_GROWTH_FACTOR=1.25
 addons:
   postgresql: '9.4'
-install: bin/setup
+install: 
+  - sudo bin/qpid_install.sh
```

Comment:
Prefer putting this in a `before_install:` section (that section is where dependencies are usually installed, which this is).